connectors/class-connector-posts.php

```diff
@@ -255,7 +256,7 @@ public function callback_transition_post_status( $new, $old, $post ) {
 			);
 		}
 
-		if ( 'auto-draft' === $old && 'auto-draft' !== $new ) {
+		if ( in_array( $old, $start_statuses, true ) && ! in_array( $new, $start_statuses, true ) ) {
```

Comment:
@dero Does this resolve #1176? This will update the action to be `created` when the old status is `new`, `auto-draft`, or `inherit`.

Reply:
I've also update the tests to confirm that changes :+1:

Reply:
@kidunot89 This looks great, thanks!